### ISSUE TITLE
Consider other source image formats

### ISSUE THREAD

**Issue by jpstroop** [loris-imageserver/loris#29]
e.g. JPEG, TIFF and so support different libraries for format conversion. Maybe this should be abstracted out of the Loris class, so that any function that implements a common signature can be called.

**Comment by ksclarke** [comment]
Just curious... closed because it was implemented or because it was deemed unnecessary?

**Comment by jpstroop** [comment]
Sorry...I was just knocking these off because I haven't been closing them as I worked on what basically wound up amounting to a rewrite, and the branches ceased to line up nicely with the issues.

The answer is because it was implemented.

As of 0.1.0beta, Loris supports TIFF and JPEG as well as JP2 as source formats. These are implemented as what I called "Transformer" classes in the [transformations module](https://github.com/pulibrary/loris/blob/development/loris/transforms.py) and can be changed or extended to fit local needs, different library preferences, etc. in what I think is a pretty lightweight way: you just have to tell the app about them in the [config file](https://github.com/pulibrary/loris/blob/development/etc/loris.conf) (see the the sections named "transforms.*") and the classes are instantiated dynamically and passed the values of any keys in that section. The 'info.json' service will change to report what derivative formats are supported, etc. based on the implementation.
